layouts/shortcodes/observability_pipelines/destination_env_vars/chronicle.en.md

```diff
@@ -1 +1 @@
-Enter the Google Chronicle endpoint URL. For example, `https://chronicle.googleapis.com`.
\ No newline at end of file
+Enter the Google SecOps endpoint URL. For example, `https://chronicle.googleapis.com`.
```

Comment:
Haha, Google hasn't changed the endpoint URL yet?

Reply:
Thanks for reviewing @urseberry! They haven't: https://cloud.google.com/chronicle/docs/reference/rest#service-endpoint. They also haven't updated their docs url: https://cloud.google.com/chronicle/docs) 😄